core/platform/logging/logging_service_test.py

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for logging_service."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.platform.logging import logging_service
+from core.tests import test_utils
+
+
+class LoggingServicesUnitTests(test_utils.AppEngineTestBase):
+    """Tests for logging_service."""
+
+    def test_logging_services(self):
+        logging_service.enable_logging()
```

Comment:
@vojtechjelinek This is a minimal test. the enable logging function doesn't return anything thus I am not sure how to test it further.

Reply:
Can you stub `logging.Client` and then test that we call all the relevant functions?